.travis.yml

```diff
@@ -0,0 +1,28 @@
+sudo: false
+git:
+  depth: 3
+language: java
+env:
+  global:
+  - TERM=dumb         # for gradle
+matrix:
```

Comment:
Do we have to use matrix? Could we instead just use the simple config option?

```
jdk:
  - openjdk8
```

Reply:
Sure.  I had added `openjdk11` to the matrix initially but hit some strange gradle compilation failures.